src/main/java/org/springframework/integration/kafka/inbound/KafkaMessageSource.java

```diff
@@ -153,10 +153,40 @@
 	 *
 	 * @param consumerFactory the consumer factory.
 	 * @param topics the topics.
-	 * @see #KafkaMessageSource(ConsumerFactory, KafkaAckCallbackFactory, boolean, String...)
+	 * @see #KafkaMessageSource(ConsumerFactory, ConsumerProperties, KafkaAckCallbackFactory, boolean)
 	 */
+	@Deprecated
 	public KafkaMessageSource(ConsumerFactory<K, V> consumerFactory, String... topics) {
-		this(consumerFactory, new KafkaAckCallbackFactory<>(), false, topics);
+		this(consumerFactory, new ConsumerProperties(topics), new KafkaAckCallbackFactory<>(), false);
+	}
+
+	/**
+	 * Construct an instance with the supplied parameters. Fetching multiple
+	 * records per poll will be disabled.
+	 *
```

Comment:
No blank line in the method JavaDocs, please: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting

Reply:
Okay, but I think the lint didn't warn about it.

Reply:
Well, we don't talk about some tool. It is just a matter of Spring best practice for code style.